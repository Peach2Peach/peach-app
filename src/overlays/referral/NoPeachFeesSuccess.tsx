import { ReactElement } from 'react'
import { Text } from '../../components'
import i18n from '../../utils/i18n'

export const NoPeachFeesSuccess = (): ReactElement => <Text>{i18n('settings.referrals.noPeachFees.popup.success')}</Text>
