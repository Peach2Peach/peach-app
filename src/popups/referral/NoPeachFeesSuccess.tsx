import PeachText from '../../components/text/Text'
import i18n from '../../utils/i18n'

export const NoPeachFeesSuccess = () => <PeachText>{i18n('settings.referrals.noPeachFees.popup.success')}</PeachText>
