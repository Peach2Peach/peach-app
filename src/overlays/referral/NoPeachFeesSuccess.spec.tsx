import { createRenderer } from 'react-test-renderer/shallow'
import { NoPeachFeesSuccess } from './NoPeachFeesSuccess'

describe('NoPeachFeesSuccess', () => {
  const shallowRenderer = createRenderer()
  it('should render correctly', () => {
    shallowRenderer.render(<NoPeachFeesSuccess />)
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot()
  })
})
