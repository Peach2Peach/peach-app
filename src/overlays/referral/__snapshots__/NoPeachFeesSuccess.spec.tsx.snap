// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPeachFeesSuccess should render correctly 1`] = `
<PeachText>
  You successfully activated this powerup. Let's getÂ trading!
</PeachText>
`;
